package.json

```diff
@@ -37,7 +37,7 @@
     "postgenerate": "yarn lint:write",
     "preinstall": "yarn config set ignore-engines true",
     "postinstall": "git submodule update --init --recursive && patch-package",
-    "build": "yarn build:libs --filter='!@account-kit/react-native-signer' --filter='!@account-kit/react-native-signer-example'",
+    "build": "yarn build:libs --filter='!@account-kit/react-native-signer' --filter='!@account-kit/react-native-signer-example' --filter='!examples/react-native-bare-example'",
```

Comment:
this probably isn't necessary since `build:libs` already avoids building anything in examples